### ISSUE TITLE
Please release a new version that doesn't break with clang 16

### ISSUE THREAD

**Issue by bugnano** [cloudflare/boring#124]
Clang 16 introduced a breaking change, and bindgen needs to be updated accordingly.
Given that version 2.1.0 requires the older bindgen version, the bindgen-sys compilation fails with clang 16.
The updated bindgen dependency is already in the latest git, so it's only a matter of releasing a new version.

**Comment by makindotcc** [comment]
I've made PR for that and it works on my clang version ``16.0.5 (Fedora 16.0.5-1.fc38)``. (https://github.com/cloudflare/boring/pull/125)
```toml
boring = { git = "https://github.com/makindotcc/boring", branch = "update-bindgen" }
```
thank you, you saved me a lot of pain just by saying it is clang16/outdated bindgen version issue.

**Comment by inikulin** [comment]
fixed in 3.0.0
